bin/run-browser-test-env

```diff
@@ -12,45 +12,73 @@ if [ -f ".secrets" ]; then
   . .secrets
 fi
 
+ci="no"
+
 # Get cloud provider
 _setArgs(){
   while [ "${1:-}" != "" ]; do
     case "$1" in
+      "--ci")
+        ci="yes"
+        ;;
+
       "--azure")
-       cloudProvider="azure"
-       emulator="azurite"
-       devServer="http://localhost:10000"
-       export STORAGE_SERVICE_NAME="azure-blob"
-       ;;
+        readonly CLOUD_PROVIDER="azure"
+        readonly EMULATOR="azurite"
+        readonly SERVER_URL="http://localhost:10000"
+        export STORAGE_SERVICE_NAME="azure-blob"
+        ;;
+
       "--aws")
-       cloudProvider="aws"
-       emulator="localstack"
-       devServer="http://localhost:6645"
-       export STORAGE_SERVICE_NAME="s3"
+        readonly CLOUD_PROVIDER="aws"
+        readonly EMULATOR="localstack"
+        readonly SERVER_URL="http://localhost:6645"
+        export STORAGE_SERVICE_NAME="s3"
        ;;
     esac
+
     shift
   done
 }
 
 _setArgs "$@"
 
 # default to AWS for now
-if [ -z "$cloudProvider" ]; then
-  cloudProvider="aws" && emulator="localstack" && devServer="http://localhost:6645"
+if [ -z "${CLOUD_PROVIDER}" ]; then
```

Comment:
We can remove this duplication by setting these above the case on 20 as the defaults and have it explicitly at the start and remove the aws block.

Reply:
done